<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="MainWidget" native="1" name="mainWidget" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenRom" />
    <addaction name="actionCloseRom" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Emulation" >
    <property name="title" >
     <string>&amp;Emulation</string>
    </property>
    <addaction name="actionStart" />
    <addaction name="actionPause" />
    <addaction name="actionStop" />
    <addaction name="separator" />
    <addaction name="actionSaveState" />
    <addaction name="actionSaveStateAs" />
    <addaction name="actionLoadState" />
    <addaction name="actionLoadStateFrom" />
    <addaction name="separator" />
    <addaction name="actionCurrentSaveStateSlot" />
   </widget>
   <widget class="QMenu" name="menu_Settings" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionFullScreen" />
    <addaction name="actionConfigureMupen64Plus" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Emulation" />
   <addaction name="menu_Settings" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenRom" />
   <addaction name="separator" />
   <addaction name="actionStart" />
   <addaction name="actionPause" />
   <addaction name="actionStop" />
   <addaction name="separator" />
   <addaction name="actionLoadState" />
   <addaction name="actionSaveState" />
   <addaction name="actionCurrentSaveStateSlot" />
   <addaction name="separator" />
   <addaction name="actionFullScreen" />
  </widget>
  <action name="actionOpenRom" >
   <property name="text" >
    <string>&amp;Open Rom</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseRom" >
   <property name="text" >
    <string>&amp;Close Rom</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="text" >
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="text" >
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>S&amp;top</string>
   </property>
   <property name="shortcut" >
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSaveState" >
   <property name="text" >
    <string>Sa&amp;ve State</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F7</string>
   </property>
  </action>
  <action name="actionSaveStateAs" >
   <property name="text" >
    <string>Save State &amp;as...</string>
   </property>
  </action>
  <action name="actionLoadState" >
   <property name="text" >
    <string>&amp;Load State</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionLoadStateFrom" >
   <property name="text" >
    <string>L&amp;oad State from...</string>
   </property>
  </action>
  <action name="actionCurrentSaveStateSlot" >
   <property name="text" >
    <string>&amp;Current Save State Slot...</string>
   </property>
  </action>
  <action name="actionFullScreen" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Full Screen</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionConfigureMupen64Plus" >
   <property name="text" >
    <string>&amp;Configure Mupen64Plus</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header>mainwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
